dev-requirements.txt

```diff
@@ -3,3 +3,4 @@ pytest-cov
 factory-boy>=2
 mock
 ckantoolkit
+assertpy
```

Comment:
I'm against adding this dependency. Pytest as the docs suggest works very smart with native python assertions (please look at https://docs.pytest.org/en/7.1.x/how-to/assert.html). Vanilla python asserts is the usual way to assert in tests not only in the rest of our code base but in many other python projects.
assertpy doesn't look like actively maintained project and the added readability is purly subjective. Good local variable naming and simple `==`, `=!`, `>` assertions are easy to read and understand both in code and test error reports.